python/ray/data/_internal/util.py

```diff
@@ -156,18 +156,31 @@ def _autodetect_parallelism(
     if parallelism < 0:
         if parallelism != -1:
             raise ValueError("`parallelism` must either be -1 or a positive integer.")
+
+        if (
+            ctx.min_parallelism is not None
+            and ctx.min_parallelism != DEFAULT_READ_OP_MIN_NUM_BLOCKS
+        ):
+            logger.warning(
+                "``DataContext.min_parallelism`` is deprecated in Ray 2.10. "
+                "Please specify ``DataContext.read_op_min_num_blocks`` instead."
+            )
+
```

Comment:
Should we set `ctx.read_op_min_num_blocks = ctx.min_parallelism` so that `DataContext.min_parallelism` continues to work?

Reply:
Yes, sorry missed that part.

Reply:
Updated.